pkg/docker/client.go

```diff
@@ -36,10 +36,12 @@ const (
 // The Docker client can be direclty used for some functions not provided by this manager.
 type DockerManager struct {
 	// Client represets the Docker client.
-	Client *docker_client.Client
+	Client     *docker_client.Client
+	EndPoint   string
+	AuthConfig *docker_client.AuthConfiguration
 }
 
-func NewDockerManager(endpoint string) (*DockerManager, error) {
+func NewDockerManager(endpoint, registryServer, registryUsername, registryPassword string) (*DockerManager, error) {
```

Comment:
What's the difference between `endpoint` and `registryServer`?

Reply:
`endpoint` is Docker server endpoint, which is used to new Docker client.
`registryServer` is the registry server.